### DISCUSSION TITLE
Cannot dump Romeow and Julicat for Sega Mega Drive

### DISCUSSION THREAD

**Discussion by sprudel79** [sanni/cartreader#819] [Need help]
Hi everyone, 
please help with the mentioned game.
Here's the output of the dump attempt:

```
OSCR HW5 V12.4

[+] PC Engine/TG16

[+] WonderSwan (3V)

[+] PC Engine/TG16

[+] WonderSwan (3V)

[+] Mega Drive/Genesis

[+] Game Cartridge


Cart Info
 
Name: ROMEOWJULICAT
bramCheck: 0000
Size: 8 MBit
ChkS: 0000
Sram: 2 KBit
 
Press Button...

[+] Read Rom


Saving to MD/ROM/ROMEOWJULICAT/20/...
[*******************]
Internal checksum...Error
0B0A

CRC32... 38BF19E6 -> Not found
```
Starting the produced file will show graphical garbage.
Thanks in advance!

**Comment by PsyK0p4T** [comment]
Hi. 
Getting these errors during dumping is "normal" for now. Internal checksum was left blank in header by the devs, and final CRC32 is currently unknown/not supported. I can try to help with test scripts, are you on our Discord server ?

**Comment by sprudel79** [comment]
Thanks @PsyK0p4T  for your quick response, I am not a member yet of your Discord server and I appreciate your help. What am I supposed to do?

**Reply by PsyK0p4T** [reply]
You can join us by using the PC/mobile app, or a web browser : https://github.com/sanni/cartreader/discussions/555
